src/material-experimental/mdc-core/_core-theme.scss

```diff
@@ -28,6 +29,17 @@
           @include elevation.private-theme-elevation($zValue, $color);
         }
       }
+
+      // Wrapper element that provides the theme background when the user's content isn't
+      // inside of a `mat-sidenav-container`. Note that we need to exclude the ampersand
+      // selector in case the mixin is included at the top level.
+      .mat-mdc-app-background#{if(&, ', &.mat-mdc-app-background', '')} {
+        $background: map.get($color, background);
```

Comment:
Note that here I'm going through our own theming system, rather that MDC's, because I couldn't figure out how to generate the same colors with MDC. The only thing remotely close is the `filled` appearance of the form field, but its background color on a dark theme was a bit lighter than what we have.